roles/iridl/tasks/main.yaml

```diff
@@ -308,6 +308,21 @@
   when: run_update_script | bool
   register: update_script
 
+- set_fact:
+    update_python_maproom_script: /usr/local/bin/update_python_maproom_data
```

Comment:
The name of this variable is misleading--could be interpreted to mean that it's the script that rebuilds the application. `update_python_maproom_data_script` is kind of long, but I can't think of anything shorter that seems appropriate.

Reply:
On second thought, not worth worrying about. The only place the variable is used is immediately after it's declared. Not much risk of confusion.